style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: purple;
+}
```

Comment:
- [ ] Kindly make sure the `style.css` file includes one class with styles for your header as required for the project. I suggest you add one class in the `<h1>` tag(in the index.html file) and then use it to style your header.